### PR TITLE
Update spacing sizes control metrics and icons

### PR DESCRIPTION
**Pull request by richtabor**
## What?

- Improve the icons so they are more visible (higher contrast and not blurry on high-resolution screens) and a bit smaller, to fit better in the Inspector. The color is technically darker than the track value, but optimally it does not look so (as it's smaller visually). 
- Condenses a bit as well, so the controls take up less space — but are still approachable. 
- Adjusts the icons iso they are in line with the label. 
- Small refinement to the settings icon. 

## Testing Instructions

1. Open a post or page.
2. Insert a group block. 
3. Open the block inspector.
4. See Dimensions panel. 
5. Add various padding and margin tools. 

## Screenshots or screencast 

| Before  | After |
| ------------- | ------------- |
|CleanShot 2023-09-14 at 13 18 48CleanShot 2023-09-14 at 13 18 34CleanShot 2023-09-14 at 13 19 35CleanShot 2023-09-14 at 13 29 03|CleanShot 2023-09-14 at 13 20 14CleanShot 2023-09-14 at 13 20 59CleanShot 2023-09-14 at 13 22 25CleanShot 2023-09-14 at 13 28 57|

### CODE REVIEW

**Comment by ndiego** [comment]
This is looking great 💪

**Review by apeatling — APPROVED** [review]
Lots of good refinements here, works well. :+1:

**Comment by jasmussen** [comment]
Yep, I think this is cleaner. I do think @jameskoster had some instincts in mind when proposing the original crossed icon, worth just checking the history to make sure we're not missing anything.

**Comment by jameskoster** [comment]
Nice change! The crossed icon was something to revisit that we never got around to 🙈 I still think it would be useful for the axial icon to differ from the custom one, but only if we can find a compelling design. Otherwise, in practise it may be fine for them to be the same.

File: packages/icons/src/library/sides-bottom.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ import { SVG, Path } from '@wordpress/primitives';
 
 const sidesBottom = (
 	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-		<Path d="M7 20.5h10V19H7v1.5z" style={ { fill: '#1e1e1e' } } />
 		<Path
-			d="M3.5 17H5V7H3.5v10zM19 7v10h1.5V7H19zM7 5h10V3.5H7V5z"
-			style={ { fill: '#1e1e1e', opacity: 0.1 } }
+			d="m7.5 6h9v-1.5h-9zm0 13.5h9v-1.5h-9zm-3-3h1.5v-9h-1.5zm13.5-9v9h1.5v-9z"
+			style={ { fill: '#1e1e1e', opacity: 0.25 } }
```
Comment (jameskoster) [inline packages/icons/src/library/sides-bottom.js]: Do we need to specify `fill` on the icon? If so would it make sense to use `currentColor` in preparation for admin themes?

Comment (richtabor) [reply packages/icons/src/library/sides-bottom.js]: Good call. Probably not!

**Comment by richtabor** [comment]
> I still think it would be useful for the axial icon to differ from the custom one, but only if we can find a compelling design. Otherwise, in practise it may be fine for them to be the same.

I tried more of the same, but it wasn't looking/feeling right. I think the sides icon works in both places. Feels more like a "other padding options" icon, rather than a "horizontal and vertical padding is enabled" icon — which I think is ok.

File: packages/icons/src/library/sides-bottom.js (hunk 1 above)
Comment (richtabor) [reply packages/icons/src/library/sides-bottom.js]: Resolved in https://github.com/WordPress/gutenberg/pull/54470/commits/b825970e5cac8ae010bf9ccc91aa15634bce74a4.

**Review by jameskoster — APPROVED** [review]

**Pull request merged**